src/src/WebServer/HTML_wrappers.cpp

```diff
@@ -312,7 +312,7 @@ void html_add_script(bool defer) {
   if (defer) {
     addHtml(F(" defer"));
   }
-  addHtml(F(">"));
+  addHtml(F('>'));
```

Comment:
The F macro can only be applied to strings not on chars, this line should be: `addHtml('>');` (that's why the build failed)